cmd/thriftrw-list-deps/main_test.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"path/filepath"
+	"testing"
+
+	"sort"
+
+	"io/ioutil"
+
+	"os"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestThriftrwListDeps(t *testing.T) {
+	tmpDir, err := ioutil.TempDir("", "")
+	if err != nil {
+		t.Skipf("error creating temporary directory: %v", err)
```

Comment:
This should be a `require.NoError`. That will stop execution and mark the test
as failed. Otherwise, the `go test` will succeed and the test will be skipped.